### PR TITLE
Update Button docs for Hover

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
Updates the docs for Button hover Indicator 
#### Where should the reviewer start?
button/doc.js
#### What testing has been done on this PR?
none
#### How should this be manually tested?
none
#### Any background context you want to provide?
The issue was brought up that button hover indicator did not 
have a good explanation in the docs I wanted to provide an example for 
each of the props as well as add a little more detail to the documents
#### What are the relevant issues?
issue #3330 
#### Do the grommet docs need to be updated?
auto 
#### Should this PR be mentioned in the release notes?
no
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

File: src/js/components/Button/README.md [hunk 1]
```diff
@@ -160,19 +160,21 @@ boolean
 **hoverIndicator**
 
 The hover indicator to apply when the user is mousing over the
-button. An object can be also be specified for color index support:
-{background: 'neutral-2'}. This prop is meant to be used only
-with plain Buttons.
+button. HoverIndicator can accept a boolean, string, and object
```
Comment (ShimiSun) [inline src/js/components/Button/README.md]: This is repeating the statement of types on lines 172-174.

File: src/js/components/Button/README.md [hunk 2]
```diff
@@ -160,19 +160,21 @@ boolean
...
+Where the value of true or "background" applies theme.global.hover.background;
+a string specifying a color (e.g. '#999999'; 'neutral-1') Or, a 'url()' for an image;
+an object specifying color index support: (e.g.
+{color: 'neutral-2'}, {"dark": "light-1", "light": "dark-2"}).
```
Comment (ShimiSun) [inline src/js/components/Button/README.md]: This is not specific for hoverIndicator, so I don't think it is the right place.

Comment (britt6612) [reply src/js/components/Button/README.md]: Agree we can move this to one wiki page will start working on that for users!

**Comment by ShimiSun** [comment]
Color properties wiki could be found here https://github.com/grommet/grommet/wiki/Color-Properties. Closing this PR.

**Pull request closed without merging**